### PR TITLE
[potrace] Added types for potrace

### PR DESCRIPTION
**Pull request by proohit**
Adds type declarations for [potrace](https://www.npmjs.com/package/potrace). I am not the author of `potrace`. I stumbled over missing type declarations when using `potrace` with TS.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
Note: I was not able to install the type declarations from this repository. I have a project that I could test it on, if I know how to install the types from here.
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration` 
Note: There are jsDoc declarations, but using them for generation throws errors on my end
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

Some advice regarding the open points above is very much appreciated.

### CODE REVIEW

File: types/potrace/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "esModuleInterop": true,
```
Comment (peterblazejewicz) [inline types/potrace/tsconfig.json:11]:
```suggestion
        "strictNullChecks": true
```
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

File: types/potrace/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
+// Type definitions for potrace 2.1
+// Project: https://github.com/tooolbox/node-potrace
+// Definitions by: Direnc Timur <https://github.com/proohit>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import 'node';
```
Comment (peterblazejewicz) [inline types/potrace/index.d.ts:6]:
```suggestion
/// <reference types="node" />
```
this is established pattern here to reference Node types

Comment (proohit) [reply types/potrace/index.d.ts]: fixed in  2217278

File: types/potrace/tsconfig.json (hunk 1 above)
Comment (proohit) [reply types/potrace/tsconfig.json]: fixed in 2217278

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@proohit thx!

**Comment by proohit** [comment]
Ready to merge

**Pull request merged**